oci-library/oci-hol/java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -284,12 +324,52 @@ Install Management Agent (If your host is Windows, skip to **For Windows** Secti
 
   ![image of modify agent settings page](/../images/fleet-modify-agent-settings-new.png)
 
+## Task 6: Verify application logging by Java Usage Tracker
+1. For the logging of applications to be visible, Java applications must be run again after the installation of the Management Agent. Run the following commands to run the HelloWorld application again:
+
+    ```
+    <copy>
+    javac HelloWorld.java
+    java HelloWorld
+    </copy>
+    ```
+
 11. If tagging and installation of management agents is successful, Java Runtimes will be indicated on the Fleet Main Page.
 
   ![image of successful installation](/../images/successful-installation.png)
 
+12. Select your fleet and click **Applications** under **Resources**. You should now see two applications, one of which is from the javac compiler command and the other from the HelloWorld application.
+
+  ![image of applications after successful installation](/../images/successful-installation-applications.png)
+
+  You may now **proceed to the next lab.**
+
 ## Troubleshoot Management Agent Installation Issues
 
+**For Task 1**
+* If you are unable to download the management agent software using `wget`, you may download the software from the Oracle Cloud Console.
+* First, open the navigation menu, click **Observability & Management**, and then click **Fleets** under **Java Management**. Select the fleet that you have created.
+  ![image of console navigation to java management service](/../images/console-navigation-jms.png)
+
+* Click **Set Up Management Agent**.
+  ![image of fleet details page](/../images/fleet-details-page.png)
+* Click **Download management agent software**.
+  ![image of set up management agent page](/../images/fleet-set-up-management-agent.png)
+* Open up a **Terminal** or **Command Prompt** window in the local machine where the management agent software file is saved.
+* Enter the following command to transfer the management agent software file via scp into the remote host compute instance.
+
+    ```
+    <copy>
+    scp <full_path_of_file_to_be_transfered_on_local_host> opc@<public_IP_Address>:<full_path_of_remote_directory_transferred_to>
```

Comment:
typo on transfered. recommend using a spell checker extension on vscode to check through once